main.tf

```diff
@@ -156,11 +156,16 @@ module "openshift_install" {
   ocp_virtualization_enable = var.ocp_virtualization_enable
 }
 
+
 resource "null_resource" "get_kubeconfig" {
   depends_on = [module.prepare_openshift.finished]
 
   provisioner "local-exec" {
-    command = "mkdir -p ${path.root}/auth; scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${module.sshkey.ssh_private_key_file} root@${module.bastion.lb_ip}:/tmp/artifacts/install/auth/* ${path.root}/auth/"
+    command = <<EOT
+      [[ -d ${path.root}/auth ]] || mkdir -p ${path.root}/auth
```

Comment:
without the `[[ -d ... ]]` I was getting errors that suggested `mkdir -p` was returning an exit status when the directory existing that triggered a failure. I initially has `|| true` but this approach does not hide actual errors.